tests/end2end/cypress/integration/form_advanced-ghaction.js

```diff
@@ -67,9 +68,11 @@ describe('Advanced form', function () {
 
         // Assert quartier value is good for another drawn point
         cy.get('#map').click(600, 350)
-        cy.get('#jforms_view_edition_quartier option').should('have.text', 'MONTPELLIER CENTRE')
+        cy.get('#jforms_view_edition_quartier option').should('have.length', 2)
+        cy.get('#jforms_view_edition_quartier option').last().should('have.text', 'MONTPELLIER CENTRE')
+        cy.get('#jforms_view_edition_quartier').select('MC')
```

Comment:
@rldhont In QGIS user does not have to select. Could we have the same behavior in Lizmap?
![Peek 17-02-2022 20-38](https://user-images.githubusercontent.com/2145040/154558045-7cf5ec41-813e-4e26-b892-e75aa241575d.gif)